.gitignore

```diff
@@ -1,3 +1,4 @@
 dist
 node_modules
-build/
```

Comment:
Do we need `build/` anymore? The next.js build will go to `dist` and our own build goes to `ausguck` so i guess we can remove this?